lib/cfd_trading/cfd_order_confirmation.dart

```diff
@@ -13,6 +12,8 @@ import 'package:ten_ten_one/utilities/tto_table.dart';
 
 import 'package:ten_ten_one/ffi.io.dart' if (dart.library.html) 'ffi.web.dart';
 
+import '../utilities/async_button.dart';
```

Comment:
🔧 Please use package imports, that will allow us to move the button to another directory without having to update this file.